src/thematic/4_vegetation/sfp-401_vegopname-terrest_nl/index.Rmd

```diff
@@ -10,6 +10,11 @@ params:
   version_number: "2021.00.dev"
   language: nl
   theme: vegetation
+  width_OV_m: 3
+  length_OV_m: 3
+  width_B_m: 16
+  length_B_m: 16
```

Comment:
The preferred naming convention is so-called snake_case words. This means everything lower case and words separated by a "_". So width_OV_m would become width_ov_m.